bin/utils/migrationsLister.js

```diff
@@ -0,0 +1,37 @@
+const color = require('colorette');
+
+const { success } = require('./cli-config-utils');
+
+function migrationsLister(completed, newMigrations) {
```

Comment:
Sorry for the constant stream of requests. Could you please rename function to be a verb - "listMigrations"? File name is fine.

Reply:
No Problem at all.
Will rename and push the changes.